workshops/read_the_internet/README.md

```diff
@@ -0,0 +1,175 @@
+
+I will just use the second row and first column for this example
+```python
+tableCols = tableRows[1].findAll('td')
+tableCol = tableCols[0]
+tableCol.text
+```
+This returns `'\r\n          TOTAL\r\n        '` with weird spacing and newline and indents. With Python we can use the `strip` function on the string and we will end with this code `tableCol.text.strip()` and this text `'TOTAL'`. Now that we got that handled lets get to reading the data!
+
+First we have a have a for each loop to loop through every row in the table. You make notice the `[1:]`, this was added to make the loop just a little more effcient because the first row at index `[0]` is just the table header with no data. Next we use an if statment to check if the current row's country is the United States (This can be replaced by any country). **Do Note** that we use the `.find()` function because we are only checking the first column and `.text[2:]` because the website has a weird astricks we need to remove. Once we find the country of our choice, we use `findAll()` to get all the columns in the row and print out some data.
+```python
+for row in tableRows[1:]:
+    if 'United States' == row.find('td').text[2:].strip():
+        columns = row.findAll('td')
+        print(columns[0].text[2:].strip())
+        print(f'Confirmed Cases: {columns[1].text.strip()}')
+        print(f'New Cases: {columns[3].text.strip()}')
+        print(f'Critical Cases: {columns[5].text.strip()}')
+```
+The output at the time of writing this workshop is:
+```
+United States
+Confirmed Cases: 9,566,690
+New Cases: 28,845
+Critical Cases: 88,127
+```
+Scary Numbers! Hopefully at the time of reading this workshop things won't be too much worse :(
+
+Here is some example code for checking for multiple countries
+```python
+countries = ['United States', 'Spain', 'Canada']
+for row in tableRows[1:]:
+    if row.find('td').text[2:].strip() in countries:
+        columns = row.findAll('td')
+        print(columns[0].text[2:].strip())
+        print(f'Confirmed Cases: {columns[1].text.strip()}')
+        print(f'New Cases: {columns[3].text.strip()}')
+        print(f'Critical Cases: {columns[5].text.strip()}\n')
+```
+
+### Thats it! You can now get live COVID19 data but more importantly you are now primed with knowledge to webscrape anything!
+![Sponge bob done gif](https://cloud-5bmb4t5tp.vercel.app/0completed.gif)
+You can check out more advanced/in depth projects here. In this project you will be shown how to use selenium to webscrape that will allow you to webscrape pages like amazon or google that use ajax to load as you scroll (https://www.analyticsvidhya.com/blog/2020/08/web-scraping-selenium-with-python/). Check out this workshop by realpython that goes more in depth and links to other resources that can allow you to webscrape more complicated websites (https://realpython.com/beautiful-soup-web-scraper-python/).
```

Comment:
This should not be dumped all in one paragraph one sentence after another. Instead, you should start a new section called "Hacking" and inside the section include each of these things in a list